### DISCUSSION TITLE
Customer Password

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [uvdesk/community-skeleton#585] [Q&A] [answered]
Hello,

I hope you're well and everything is good for you.

How to set or generate passwords for customers I created?

Thank you very much.
Best regards,

**Comment by komal-sh-27** [comment, marked as answer]
@YunusEmreAlps 

Please do the following steps to set your customer password:

- Setup Swiftmailer and then Email Settings, which enable all outgoing mail from the support system:
For setup swiftmailer and the email settings follow the guide from [here](https://www.uvdesk.com/en/blog/setup-mailbox-uvdesk-open-source-helpdesk/)

- Then create the customer or agent, who will receive mail to set a password

**Reply by YunusEmreAlps** [reply]
Hi @Komal-sharma-2712 

I get an error when I follow the steps in the documentation.

![Screenshot (500)](https://user-images.githubusercontent.com/44679555/182148969-27d38673-1394-4c57-8c09-63f69150706e.png)
![Screenshot (501)](https://user-images.githubusercontent.com/44679555/182148974-e6718e21-7bb0-4872-a54a-1144fa972ffa.png)
![Screenshot (497)](https://user-images.githubusercontent.com/44679555/182148509-467d2190-b9fb-48e6-852e-d771ee7cf821.png)
![Screenshot (498)](https://user-images.githubusercontent.com/44679555/182148517-ad88ee07-c387-4af0-aed2-54ae5849aaa1.png)
![Screenshot (499)](https://user-images.githubusercontent.com/44679555/182148518-8a993460-54c5-4878-a9a7-5fa6833ab46c.png)

**Reply by komal-sh-27** [reply]
Let us know you are using in the password field, which password uses a normal password or an app password.

If you are using the normal password in the swiftmailer and mailbox settings please used the app password not use normal password:


![image](https://user-images.githubusercontent.com/63539748/182151592-15ced348-a9b3-4198-aa97-a57bc9d61bd9.png)


![image](https://user-images.githubusercontent.com/63539748/182153324-982fbfdb-eaa1-4abd-a50f-471d0abce08e.png)


![image](https://user-images.githubusercontent.com/63539748/182153499-d62077c6-a6d9-4dee-ab03-76ef76284401.png)


To create an app password follow this guide from [here](https://in.help.yahoo.com/kb/SLN15241.html#:~:text=Generate%20an%20app%20password,Click%20Generate%20password.)

**Reply by YunusEmreAlps** [reply]
Thank You @Komal-sharma-2712 for your guide.
I did it! I dit it!

**Reply by PeopleInside** [reply]
@YunusEmreAlps I think the error you got is because something is not correct with email settings.
When you see the showed error in your screenshot, emails are generated or not?

You may try to insert SMTP settings without using the drop down predefined Yahoo and see if something change.
How to configure the host field is explained [here](https://www.uvdesk.com/en/blog/setup-mailbox-uvdesk-open-source-helpdesk/). 

Incoming Mail (IMAP) Server

    Transport: Select transport as Gmail, Yahoo, or IMAP
    Host: Every email service has a unique IMAP HOST like Gmail – {imap.gmail.com:993/imap/ssl}INBOX, Yahoo – 

**{imap.mail.yahoo.com:993/imap/ssl}INBOX**

    Email: Put email address of IMAP(incoming mail server)
